test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
@destroyerofbuilds I just pushed a commit to throw an error if a version is requested for a scoped package, and update the tests to check for the error in the scoped package case.